deepsoccer_single: #namespace
    task_and_robot_environment_name: 'DeepSoccerSingle-v0'
    ros_ws_abspath: "/home/kimbring2/catkin_ws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
    
    sl_training: True
    gpu_use: True

    workspace_path: "/home/kimbring2/catkin_ws/src/my_deepsoccer_training/"
    save_path: "/home/kimbring2/catkin_ws/src/my_deepsoccer_training/human_data/"
    save_file: "deepsoccer_data_899"